### PR TITLE
Added custom validator function access to the object

### PR DESCRIPTION
**Pull request by tinchoz49**
PR for the issue #5

### CODE REVIEW

File: test/validator.spec.js [hunk 1]
```diff
@@ -218,27 +220,7 @@ describe("Test _processRule", () => {
 		expect(res[1].iterate).toBe(false);
 
 		expect(v.compile).toHaveBeenCalledTimes(1);
-		expect(v.compile).toHaveBeenCalledWith({id: "number"});		
-	});
-
-	it("should call checkWrapper & processRule if type is Array", () => {
```
[the remark this answers is not in the document]
Comment (tinchoz49) [reply test/validator.spec.js]: I don't know what I did with that test :confused:, sorry (fixed)

[the remark this answers is not in the document]
Comment (icebob) [reply test/validator.spec.js]: nop :)

**Comment by icebob** [comment]
Thank you!

File: test/validator.spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tinchoz49) [reply test/validator.spec.js]: probably is time to change from vim to vscode :smile:

[the remark this answers is not in the document]
Comment (icebob) [reply test/validator.spec.js]: VSCode is awesome ;)

**Pull request merged**